app/models/item.rb

```diff
@@ -25,6 +24,10 @@ def no_orders?
     item_orders.empty?
   end
 
+  def update_inventory(quantity)
+    update_attributes(inventory: (inventory - quantity))
+  end
```

Comment:
![](https://media3.giphy.com/media/5vYwHanb6y3HWBQIox/giphy.gif?cid=5a38a5a22cf5623c7821775861fd1cef0bef7290a9d2412e&rid=giphy.gif)

Reply:
^ might be over kill, but I do like that it is so cute and simple.